### PR TITLE
fix: kill the image compose pre text

### PR DESCRIPTION
**Pull request by BillySh1**
## Description

Closes #MF-1364

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Review by guanbinrui — APPROVED** [review]

**Comment by guanbinrui** [comment]
@BillySh1 Please merge to `release/2.10.0`.

File: packages/mask/shared-ui/locales/en-US.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
     "additional_post_box__encrypted_post_pre_ito": "$t(promote_ito) {{encrypted}}",
     "additional_post_box__encrypted_post_pre_file_service_twitter_official_account": "$t(promote_file_service)\n {{encrypted}}",
     "additional_post_box__encrypted_post_pre_file_service": "$t(promote_file_service) {{encrypted}}",
-    "additional_post_box__steganography_post_pre": "This image is encrypted with #mask_io.\n{{random}}\n$t(promote)",
```
Comment (Jack-Works) [inline packages/mask/shared-ui/locales/en-US.json:69]: If you remove this `random`, sending similar pictures might be blocked by Facebook.

**Review by Jack-Works — CHANGES_REQUESTED** [review]

File: packages/mask/shared-ui/locales/en-US.json (hunk 1 above)
Comment (BillySh1) [reply packages/mask/shared-ui/locales/en-US.json:69]: @DimensionDev/qa  close

**Comment by BillySh1** [comment]
close

**Pull request closed without merging**